src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -41,34 +49,34 @@
       <!-- List Content -->
       <v-row
         class="people-roles-content section-container"
-        :class="{
-          'summary-view': isSummaryView,
-          'invalid-section': invalidOrgPersons
```

Comment:
Don't need invalid-section here because it's already used on the div that contains this row.